OloEngine/src/OloEngine/Asset/MeshCache.cpp

```
@@ -0,0 +1,279 @@
+                if (ec)
+                {
+                    OLO_CORE_ERROR("MeshCache: Failed to clear animation cache: {}", ec.message());
+                }
+                else
+                {
+                    OLO_CORE_INFO("MeshCache: Cleared animation cache");
+                }
+            }
+        }
+
+        void InvalidateCache(const std::filesystem::path& sourcePath, const std::string& prefix)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'InvalidateCache' of similar type ('const int &') are easily swapped by mistake

(bugprone-easily-swappable-parameters)